.buildkite/auditbeat/auditbeat-pipeline.yml

```diff
@@ -3,10 +3,14 @@
 env:
   IMAGE_UBUNTU_X86_64: "family/core-ubuntu-2204"
   IMAGE_UBUNTU_ARM_64: "core-ubuntu-2004-aarch64"
```

Comment:
which kernel version does this ship with?

Reply:
It's using `5.15.0-1052-aws`

Reply:
IMAGE_UBUNTU_X86_64: Linux bk-agent-prod-gcp-1707303793816829620 6.2.0-1019-gcp #21~22.04.1-Ubuntu SMP Thu Nov 16 18:18:34 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux
IMAGE_UBUNTU_ARM_64: Linux i-095e3c2e7d81fa82d 5.15.0-1052-aws #57~20.04.1-Ubuntu SMP Mon Jan 15 17:04:12 UTC 2024 aarch64 aarch64 aarch64 GNU/Linux